app/src/main/java/com/example/plantsapp/presentation/ui/loading/LoadingDialog.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.plantsapp.presentation.ui.loading
+
+import android.app.AlertDialog
+import android.content.Context
+import com.example.plantsapp.R
+
+class LoadingDialog(context: Context) {
+
+    private val alertDialog by lazy {
+        AlertDialog.Builder(context)
+            .setView(R.layout.dialog_loading)
+            .create()
+    }
+
+    fun show() {
+        alertDialog
+            .apply {
+                show()
+                window?.setLayout(DIALOG_WIDTH, DIALOG_HEIGHT)
```

Comment:
Why is it needed?

Reply:
> Why is it needed?

it is need here to change size of the dialog window. and it only works if I put this after the show() function is called
but I guess standard size will work for now